### PR TITLE
Improve error message

### PR DESCRIPTION
**Pull request by Fatme**
http://teampulse.telerik.com/view#item/290832

### CODE REVIEW

File: lib/commands/project/create.ts [hunk 1]
```diff
@@ -31,10 +32,16 @@ export class CreateCommand extends ProjectCommandBaseLib.ProjectCommandBase {
...
-			this.$screenBuilderService.prepareAndGeneratePrompt(this.$screenBuilderService.generatorName, screenBuilderOptions).wait();
-			this.$screenBuilderService.installAppDependencies().wait();
 
-			this.$project.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
+			try {
+				this.$screenBuilderService.prepareAndGeneratePrompt(this.$screenBuilderService.generatorName, screenBuilderOptions).wait();
+				this.$screenBuilderService.installAppDependencies().wait();
+
+				this.$project.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
+			} catch(err) {
```
Comment (teobugslayer) [inline lib/commands/project/create.ts]: We can use $logger.debug(err.toString()) here

File: lib/project.ts [hunk 2]
```diff
@@ -204,7 +204,7 @@ export class Project implements Project.IProject {
 			var projectDirFiles = this.$fs.readDirectory(projectDir).wait();
 
 			if(projectDirFiles.length !== 0) {
-				this.$errors.fail("The specified directory must be empty to create a new project.");
+				this.$errors.fail("The specified '%s' directory must be empty to create a new project.", projectDir);
```
Comment (teobugslayer) [inline lib/project.ts]: I am not sure, but usually the wording is ```The specified directory '%s' must be ....```. @ikoevska, your opinion?

File: lib/commands/project/create.ts (hunk 1 above)
Comment (Fatme) [inline lib/commands/project/create.ts]: Do we need this? After deleting the directory the error is thrown.

File: lib/project.ts (hunk 2 above)
Comment (Fatme) [inline lib/project.ts]: Oo yes, it is better to be  `The specified directory '%s'`. I've changed it.

**Comment by teobugslayer** [comment]
:+1:

**Comment by tytodorov** [comment]
:+1:

**Pull request merged**